erpnext/compliance/taxes.py

```diff
@@ -61,6 +61,9 @@ def calculate_cannabis_tax(doc):
 
 
 def calculate_cultivation_tax(doc):
+	if frappe.get_cached_value("Stock Settings", None, "disable_cannabis_taxes_calculation"):
```

Comment:
@hwx remove this check from here and add it to 223 in def get_cultivation_tax(doc, items) otherwise it still calculate cultivation tax